rolemanager/role_manager_test.go

```diff
@@ -36,19 +39,111 @@ func TestRoleManager(t *testing.T) {
 
 func testSingleManager(t *testing.T, m gimlet.RoleManager) func(*testing.T) {
 	return func(t *testing.T) {
-		role1 := gimlet.Role{
-			ID:   "r1",
-			Name: "role1",
-			Permissions: map[string]int{
-				"edit": 2,
-			},
-			Owners: []string{"me"},
+		s := &RoleManagerSuite{
+			m: m,
 		}
-		assert.NoError(t, m.UpdateRole(role1))
-		dbRoles, err := m.GetRoles([]string{role1.ID})
-		assert.NoError(t, err)
-		assert.Equal(t, role1.Name, dbRoles[0].Name)
-		assert.Equal(t, role1.Permissions, dbRoles[0].Permissions)
-		assert.Equal(t, role1.Owners, dbRoles[0].Owners)
+		suite.Run(t, s)
 	}
 }
+
+type RoleManagerSuite struct {
```

Comment:
I think this is fine, but I think we should opt for using stdlib subtests rather than suites in the future.